3570/3/rating_places/app/models/comment.rb

```diff
@@ -0,0 +1,13 @@
+class Comment < ActiveRecord::Base
+  belongs_to :user
+  belongs_to :place, counter_cache: true
+
+  after_create :update_place_rating
+
+  private
+
+  def update_place_rating
+    new_rating = (place.rating * place.comments_count + rating) / (place.comments_count + 1)
```

Comment:
And what is this formula about?

Reply:
update rating after adding a new comment

Reply:
Learn about ActiveRecords methods. you don't even try to look into it

Reply:
in my case you do not have to make a request to the database

Reply:
Really? You made it in every new comment twice and after - load for display